### ISSUE TITLE
Allow for refresh at a certain interval

### ISSUE THREAD

**Issue by cjohnston1158** [bovender/indicator-ip#8]
Best I understand, there currently isn't a way to automatically refresh the IP every X seconds/minutes. It would be nice to add this.

**Comment by bovender** [comment]
The IP addresses are refreshed automatically whenever the network state changes (see #5).

Does your external IP ever change without there being a change in network state? From my understanding, with dynamic IPs from internet providers, there should be at least a short disruption in the network connection, i.e. a change in network state.

**Comment by wolfen351** [comment]
Yes, this is an issue for me too - it happens that my vpn link drops (outside of my machine's control) and I'm using this gadget to see my external ip. If I could get it to refresh every x mins, that would be great!

**Comment by wolfen351** [comment]
I managed to simulate this by using a cron job to kill and restart the indicator every 10 mins :)
